images/description/presentation-large.svg

```diff
@@ -0,0 +1,8 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <style>
+    .st0{fill:#74777B;}
```

Comment:
@njostonehouse : i usually optimize this a bit more by moving the fill to the path, ditching the `.st` class names.

@dlockhart : should we even be specifying `fill` in icons at all (or does that depend on the icon)?